Portfolio/index.html

```diff
@@ -18,7 +18,7 @@ <h1>Develop</h1>
+        <img  height="20px" width="20px" src="../phone.icon.png">

```

Comment:
Consider adding alt attributes to provide text descriptions for the images for accessibility purposes.